src/components/Match/StyledMatch.jsx

```diff
@@ -276,7 +276,7 @@ export const StyledLogFilterForm = styled.div`
 
 export const StyledDmgTargetInflictor = styled.div`
   .inflictorWithValue {  
-    img {
+    object {
```

Comment:
are we using an object tag now?

Reply:
yes, because img elements only allow text as fallback content.